src/devices/cpu/drcbex64.cpp

```diff
@@ -1484,7 +1484,7 @@ void drcbe_x64::op_nop(Assembler &a, const instruction &inst)
 
 void drcbe_x64::op_break(Assembler &a, const instruction &inst)
 {
-	a.int3();
+	smart_call_r64(a, (x86code *)(uintptr_t)&osd_break_into_debugger, rax);
```

Comment:
Do you need to put a pointer to a string for the function argument in RCX/RDI (depending on the calling convention) for the argument to `osd_break_into_debugger`?

Reply:
Yeaaah completely forgot it took a parameter. 

I ended up just implementing trampoline functions because these commands will be called very infrequently so performance isn't an issue, and it lets the compiler handle anything that it needs to. For x86 in particular it was doing stack pointer adjustments in the prologue and epilogue instead of just putting it at esp like I implemented initially, so it seems the safer choice.

Reply:
The trampoline for x86 needs to do that to keep the stack aligned because it’s a non-leaf function.  The stack will already be aligned if you haven’t done any unbalanced pushes/pops, so you can just place the value at (%esp) in the same way that `op_callc` does.